cps/pom.xml

```diff
@@ -90,6 +94,17 @@
             <artifactId>ace-editor</artifactId>
             <version>1.0.1</version>
         </dependency>
+        <dependency>
+            <groupId>com.cloudbees</groupId>
+            <artifactId>diff4j</artifactId>
+            <version>1.2</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>commons-io</groupId>
+                    <artifactId>commons-io</artifactId>
```

Comment:
Clashes with a newer version of the dependency already pulled in from Jenkins core. Maven’s dependency strategy is “closest-first” so without this exclusion you get massive linkage errors.